lib/crowdflower/job.rb

```diff
@@ -132,19 +136,19 @@ def disable_channel(channel_name)
     end
 
     def tags
-      connection.get("#{resource_uri}/#{@id}/tags") 
+      connection.get("#{resource_uri}/#{@id}/tags")
     end
 
     def update_tags(tags)
-      connection.put("#{resource_uri}/#{@id}/tags", {:body => { :tags => tags } } ) 
+      connection.put("#{resource_uri}/#{@id}/tags", {:body => { :tags => tags } } )
     end
 
     def add_tags(tags)
-      connection.post("#{resource_uri}/#{@id}/tags", {:body => { :tags => tags } } ) 
+      connection.post("#{resource_uri}/#{@id}/tags", {:body => { :tags => tags } } )
     end
 
     def remove_tags(tags)
-      connection.delete("#{resource_uri}/#{@id}/tags", {:body => { :tags => tags } } ) 
+      connection.delete("#{resource_uri}/#{@id}/tags", {:body => { :tags => tags } } )
```

Comment:
Redundant curly braces around a hash parameter.<br>Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.<br>Space inside parentheses detected.